packages/hash/shared/src/ProsemirrorSchemaManager.ts

```diff
@@ -252,7 +252,7 @@ export class ProsemirrorSchemaManager {
     /**
      * @todo we should have some concurrency here
      */
-    for (const componentId of Array.from(componentIds.values())) {
+    for (const componentId of componentIds) {
```

Comment:
Related: https://github.com/hashintel/dev/pull/442#pullrequestreview-847978355 (internal link).

We no longer have `target: es5` in Next, which was the reason for using `Array.from()` in the first place. I’m pretty sure that this code will be transpiled in Next and work in Node 14+. If I’m wrong, I will submit a reverse diff. Adding this change here to make sure we don’t forget about it later.

Reply:
Was this related to `downlevelIteration` not being allowed?